tests/editing/totals_and_components/example_data/example_test_data.csv

```diff
@@ -1,7 +1,7 @@
-reference,period,total,comp_1,comp_2,comp_3,comp_4,comp_sum,amend_total,predictive,auxiliary,abs_threshold,perc_threshold,abs_diff,perc_low,perc_high,TCC_marker,final_total,final_comp_1,final_comp_2,final_comp_3,final_comp_4
```

Comment:
Should there be a similar updated file for the thousand_pounds example csv's?

Reply:
Not needed, the thousand pounds example_test_data.csv only includes the input columns as far as I can see.

Reply:
sorry wrong file i meant to add this to example_test_data_pandas_output, my understanding is that file should now have the target_variables columns separated out and the column headers renamed, if you run sml_small/utils/pandas_example.py it should update the file